### PR TITLE
[skip ci] readme/docs update

### PR DESCRIPTION
**Pull request by vrothberg**
* Refine the "Out of scope" section regarding `docker-compose`.
* Cross reference generate/play manpages.

@conan-kudo @mheon @rhatdan @TomSweeneyRedHat PTAL

### CODE REVIEW

File: docs/podman-play-kube.1.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ kubernetes_input.yml
 the pod and containers described in the YAML.  The containers within the pod are then started and
 the ID of the new Pod is output.
 
-Ideally the input file would be one created by Podman.  This would guarantee a smooth import and expected results.
+Ideally the input file would be one created by Podman (see podman-generate(1)).  This would guarantee a smooth import and expected results.
```
Comment (Conan-Kudo) [inline docs/podman-play-kube.1.md]: Shouldn't `podman-generate-kube(1)` also be referenced in the `SEE ALSO` section?

File: docs/podman-generate-kube.1.md [hunk 2]
```diff
@@ -145,7 +147,7 @@ status:
 ```
 
 ## SEE ALSO
-podman(1), podman-container, podman-pod, podman-play
+podman(1), podman-container(1), podman-pod(1), podman-play(1)
```
Comment (Conan-Kudo) [inline docs/podman-generate-kube.1.md]: Shouldn't `podman-play-kube(1)` be referenced here?

File: docs/podman-play-kube.1.md (hunk 1 above)
Comment (vrothberg) [reply docs/podman-play-kube.1.md]: Makes sense. I changed it directly to podman-generate-kube(1). Same for play. Thanks for the suggestion!

**Pull request merged**